docs/src/1_Mechanics/04_Powers.yaml

```diff
@@ -609,13 +609,118 @@ Handy:
             - Craft > 1
             - Knowledge > 1
 
+Small Power Gain:
```

Comment:
I'm not sure why someone would take this. Under the current system max pp = sum of powers, if I spend the same 1 XP on another power, I get the same 1 PP gain plus the ability to do something extra, even if i never do it. I think that, in order to make this more attractive, it would need to add more PP than any other power for the same XP cost

Reply:
Agree with this and think we can bump up the value of these until we get feedback that they're "overpowered" or if people even take them. I've updated the Small Power Gain to add +2 PP for a 1XP cost and and Large Power Gain to add +4 PP for a 2XP cost since the base assumption is these Powers can only be taken once.